group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -543,6 +543,14 @@ public void validateOffsetCommit(
         }
     }
 
+    /**
+     * Validates the OffsetFetch request.
+     */
+    @Override
+    public void validateOffsetFetch() {
+        // Nothing.
```

Comment:
(I assume this is a todo for a future pr?)

Reply:
We will have some validation when we add the member id/epoch in the request. This is coming in the next PR.